src/plugins/blockresolver/README.md

```diff
@@ -32,43 +32,70 @@ A block consists of 2 parts:
 Currently the identifier must be unique.
  
 ## Example ##
-
-    % cat /tmp/test.block
-    bla
-    bla
-    bla
-    ### block config start
-    [section1]
-    key1 = val1
-    [section2]
-    key2 = val2
-    ### block config stop
-    asdf
-    asdf
-    asdf
-
-    % kdb mount -R blockresolver /tmp/test.block system/blocktest -c identifier="### block config"
-    % kdb export system/blocktest
-    [section1]
-    key1 = val1
-    [section2]
-    key2 = val2
-
-    % kdb set system/blocktest/section1/key12 val12
-    Create a new key system/blocktest/section1/key12 with string val12
-
-    % cat /tmp/test.block
-    bla
-    bla
-    bla
-    ### block config start
-    [section1]
-    key1 = val1
-    key12 = val12
-    [section2]
-    key2 = val2
-    ### block config stop
-    asdf
-    asdf
-    asdf
-
+```sh
+# Backup-and-Restore:system/examples/blockresolver
+sudo kdb mount -R blockresolver /tmp/test.block system/examples/blockresolver -c identifier=">>> block config" ini
+#
+# create testfile
+#
+$ echo "text" > /tmp/test.block
+$ echo "more text" >> /tmp/test.block
+$ echo "some more text" >> /tmp/test.block
+$ echo ">>> block config start" >> /tmp/test.block
+$ echo "[section1]" >> /tmp/test.block
+$ echo "key1 = val1" >> /tmp/test.block
+$ echo "[section2]" >> /tmp/test.block
+$ echo "key2 = val2" >> /tmp/test.block
+$ echo ">>> block config stop" >> /tmp/test.block
+$ echo "text again" >> /tmp/test.block
+$ echo "and more text" >> /tmp/test.block
+$ echo "text" >> /tmp/test.block
+#
+# check testfile
+#
+$ cat /tmp/test.block
+text
+more text
+some more text
+>>> block config start
+[section1]
+key1 = val1
+[section2]
+key2 = val2
+>>> block config stop
+text again
+and more text
+text
+#
+# only the block between the tags is read!
+#
+kdb export system/examples/blockresolver ini
+[section1]
+key1 = val1
+[section2]
+key2 = val2
+#
+# add a new key to the resolved block 
+#
+kdb set system/examples/blockresolver/section1/key12 val12
+#
+$ cat /tmp/test.block
+text
```

Comment:
It is not the time for big changes but to get it working ;)
(and understand the syntax you had in mind)